official/resnet/keras/keras_imagenet_main.py

```diff
@@ -187,6 +186,16 @@ def run_imagenet_with_keras(flags_obj):
                                      steps_per_epoch, samples_per_second))
 
 
+def softmax_corss_entrophy(y_true, y_pred):
+  """A cost function replicating tf's sparse_softmax_cross_entropy
```

Comment:
maybe say "loss function" instead of "cost"? 
Also I would not say replicating since we are calling TensorFlow's loss function.

Reply:
Done